packages/controllers/src/snaps/SnapController.ts

```diff
@@ -1471,3 +1556,11 @@ function isValidSnapVersionRange(version: unknown): version is string {
     typeof version === 'string' && validRangeSemver(version) !== null,
   );
 }
+
+function isLocalSnap(snapId: string): boolean {
```

Comment:
Would it make more sense to return an enum and get the snap type? So detecting if it's either local or npm in one function. Is easier to expand in the future.